akka-discovery/src/main/scala/akka/discovery/config/ConfigServiceDiscovery.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2017-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.discovery.config
+
+import akka.actor.ExtendedActorSystem
+import akka.annotation.InternalApi
+import akka.discovery.{ Lookup, ServiceDiscovery }
+import akka.discovery.ServiceDiscovery.{ Resolved, ResolvedTarget }
+import akka.event.Logging
+import com.typesafe.config.Config
+
+import scala.collection.JavaConverters._
+import scala.collection.{ breakOut, immutable }
+import scala.concurrent.Future
+import scala.concurrent.duration.FiniteDuration
+
+private object ConfigServicesParser {
```

Comment:
might still be public for Java? then add full InternalApi markers

Reply:
Yep, always under estimate the amount of publicity we get in Java